plugin/src/main/java/io/jenkins/plugins/casc/SchemaGeneration.java

```diff
@@ -132,6 +136,10 @@ private static JSONObject generateNonEnumAttributeObject(Attribute attribute) {
                 attributeType.put("type", "integer");
                 break;
 
+            case "hudson.Secret":
+                attributeType.put("type", "string");
```

Comment:
Would be nice to add a comment to the schema to indicate that this field is sensitive, but it is rather a follow-up enhancement